Tests/Pcap++Test/Tests/TcpReassemblyTests.cpp

```diff
@@ -1128,35 +1115,44 @@ PTF_TEST_CASE(TestTcpReassemblyMaxOOOFrags)
+	    true, 5, 30, 5);  // the fourth argument is the max allowed out-of-order fragments, so we only allow 5
+	pcpp::TcpReassembly tcpReassembly1(tcpReassemblyMsgReadyCallback, &results1, tcpReassemblyConnectionStartCallback,
+	                                   tcpReassemblyConnectionEndCallback, config1);
+	pcpp::TcpReassembly tcpReassembly2(tcpReassemblyMsgReadyCallback, &results2, tcpReassemblyConnectionStartCallback,
+	                                   tcpReassemblyConnectionEndCallback, config2);
 
 	std::vector<pcpp::RawPacket> packetStream;
-	PTF_ASSERT_TRUE(readPcapIntoPacketVec("PcapExamples/unidirectional_tcp_stream_with_missing_packet.pcap", packetStream, errMsg));
+	PTF_ASSERT_TRUE(
+	    readPcapIntoPacketVec("PcapExamples/unidirectional_tcp_stream_with_missing_packet.pcap", packetStream, errMsg));
 
-	for(auto iter : packetStream)
+	for (auto iter : packetStream)
 	{
 		pcpp::Packet packet(&iter);
 		tcpReassembly1.reassemblePacket(packet);
 		tcpReassembly2.reassemblePacket(packet);
 	}
 
 	pcpp::TcpReassembly::ConnectionInfoList managedConnections1 = tcpReassembly1.getConnectionInformation();
-	pcpp::TcpReassembly::ConnectionInfoList managedConnections2 = tcpReassembly2.getConnectionInformation(); // make a copy of list
+	pcpp::TcpReassembly::ConnectionInfoList managedConnections2 =
+	    tcpReassembly2.getConnectionInformation();  // make a copy of list
 	PTF_ASSERT_EQUAL(managedConnections1.size(), 1);
 	PTF_ASSERT_EQUAL(managedConnections2.size(), 1);
 	PTF_ASSERT_EQUAL(results1.flowKeysList.size(), 1);
 	PTF_ASSERT_EQUAL(results2.flowKeysList.size(), 1);
 
-	pcpp::TcpReassembly::ConnectionInfoList::const_iterator iterConn1 = managedConnections1.find(results1.flowKeysList[0]);
-	pcpp::TcpReassembly::ConnectionInfoList::const_iterator iterConn2 = managedConnections2.find(results2.flowKeysList[0]);
+	pcpp::TcpReassembly::ConnectionInfoList::const_iterator iterConn1 =
+	    managedConnections1.find(results1.flowKeysList[0]);
+	pcpp::TcpReassembly::ConnectionInfoList::const_iterator iterConn2 =
+	    managedConnections2.find(results2.flowKeysList[0]);
 	PTF_ASSERT_TRUE(iterConn1 != managedConnections1.end());
 	PTF_ASSERT_TRUE(iterConn2 != managedConnections2.end());
 	PTF_ASSERT_EQUAL(tcpReassembly1.isConnectionOpen(iterConn1->second), 1);
 	PTF_ASSERT_EQUAL(tcpReassembly2.isConnectionOpen(iterConn2->second), 1);
-	PTF_ASSERT_EQUAL(results1.stats.begin()->second.numOfDataPackets, 1); // The second data packet is incomplete so we stopped at one
-	PTF_ASSERT_EQUAL(results2.stats.begin()->second.numOfDataPackets, 7); // We hit the fragment limit so skipped the missing fragment and continued to the end
+	PTF_ASSERT_EQUAL(results1.stats.begin()->second.numOfDataPackets,
+	                 1);  // The second data packet is incomplete so we stopped at one
+	PTF_ASSERT_EQUAL(results2.stats.begin()->second.numOfDataPackets,
+	                 7);  // We hit the fragment limit so skipped the missing fragment and continued to the end
```

Comment:
nit: it'd be nicer to move the comment above the code line, that way formatting will look nicer

Reply:
I put the comment before the line, and the format looks better now.